janitor/src/main/resources/reference.conf.example

```diff
@@ -0,0 +1,30 @@
+path-to-credential = ${LEONARDO_PATH_TO_CREDENTIAL}
```

Comment:
I think we tend to not commit these? Memory is a bit hazy though, if you think it should be here that's fine. Otherwise, maybe worth an entry to .gitignore

Reply:
We used to only commit the `application.conf.example', but it turned out that the documentation on how to run the cron jobs was a bit outdated due to azure configs, and that the `reference.conf` file should also be updated with local configs, so I made these examples and updated the readme to reflect the new process.